resources/function_help/json/crs_from_text

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "crs_from_text",
+  "type": "function",
+  "groups": ["CRS"],
+  "description": "Creates a coordinate reference system from a string definition. The string can represent an auth:id code, a WKT definition, or a PROJ string definition of the CRS.",
```

Comment:
For consistency with all the other functions/variables that mentions this value, can we instead use "auth**ority**:id"  (or "Authority ID" - less developer oriented)?